rosdep/python.yaml

```diff
@@ -4706,6 +4706,9 @@ python-tk:
   arch: [python2, tk]
   debian: [python-tk]
   fedora: [python2-tkinter]
+  rhel:
+    '*': [python2-tkinter]
+    '7': [tkinter]
```

Comment:
tkinter in CentOS 7 http://rpmfind.net/linux/RPM/centos/7.8.2003/x86_64/Packages/tkinter-2.7.5-88.el7.x86_64.html